packages/space-opera/src/components/camera_settings/camera_settings.ts

```diff
@@ -106,8 +105,13 @@ export class CameraTargetInput extends ConnectedLitElement {
   @property({attribute: false}) change?: (newValue: Vector3D) => void;
   @internalProperty() target?: Vector3D;
 
+  // @ts-ignore
```

Comment:
Again, let's try to remove these whenever we see them. It's not clear which variable it pertains to.

Reply:
that one is just for the state, we need to update the target for this component, but it isn't saved in the state because we are just using the modelViewer state
pretty sure, //@ts-ignore always just ignores the next line